src/Glpi/Config/LegacyConfigurators/StandardIncludes.php

```diff
@@ -90,6 +90,13 @@ public function execute(): void
             }
         }
 
+        $is_installing = isset($_SESSION['is_installing']);
+        if ($is_installing) {
+            $skip_db_checks = true;
+            $skip_maintenance_checks = true;
+            Session::loadLanguage();
+        }
+
```

Comment:
Actually, this whole `execute()` method could probably be skipped during the install process, except for the `Session::loadLanguage()` call that may be required for the new install controller.

Reply:
The install script is inside the Kernel execution, and some other listeners need the StandardIncludes to work. I would either have to disable all listeners OR keep them here with a few tweaks, in order for installation to work.

If I add a `return` statement after `Session::loadLanguage();` here, I end up having errors for non-initialized parameters:
![image](https://github.com/user-attachments/assets/ee77118f-ec82-4789-a137-1bd3ece606d0)

I had a choice here: either add these lines in StandardIncludes, or check all places in the listeners where lack of initialization would cause issues.
I preferred to "hack" it only for installation instead of possibly breaking compatibility for other listeners, to keep BC for GLPI's existing code.

Reply:
https://github.com/glpi-project/glpi/pull/18336 should prevent this error.